jsonifier/README.rst

```diff
@@ -192,8 +192,45 @@ new requirements could be met without needing to add field or change any code.
 Note that the export values with the simple parser depends on the record's lang;
 this is in contrast with full parsers which are designed to be language agnostic.
 
+default_function parameter
+==========================
 
-NOTE: this module was named `base_jsonify` till version 14.0.1.5.0.
+In order to facilitate the usage of the many options jsonifier has, there is the 
+default_function=True options we can pass to rec.jsonify()
+this will impose as function  for all fields default_function, in order to shorten
+parser expression.
+
+default function currently retuns field: {fieldname: string_of_field, value: value} instead of
+the standard  field: value.
+
+It can be overwritten per-model or globally directly on models.py
+
+
+   Examples of default_function usage:
+
+.. code-block:: python
+   
+    # example 1 
+    parser = [('name')]
+    a.jsonify(parser=parser)
+    [{'name': 'SO3996'}]
+    >>> a.jsonify(parser=parser, default_function=False)
+    [{'name': 'SO3996'}]
+    >>> a.jsonify(parser=parser, default_function=True)
+    [{'name': {'fieldname': 'Order Reference', 'value': 'SO3996'}}]
+
+
+    # example 2
+    parser=['name', 'create_date', ('order_line', ['id' , 'product_uom', 'is_expense'])]
+    >>> a.jsonify(parser=parser, default_function=False)
+    [{'name': 'SO3996', 'create_date': '2015-06-02T12:18:26.279909+00:00', 'order_line': [{'id': 16649, 'product_uom': 'stuks', 'is_expense': False}, {'id': 16651, 'product_uom': 'stuks', 'is_expense': False}, {'id': 16650, 'product_uom': 'stuks', 'is_expense': False}]}]
+    >>> a.jsonify(parser=parser, default_function=True)
+    [{'name': {'fieldname': 'Order Reference', 'value': 'SO3996'}, 'create_date': {'fieldname': 'Creation Date', 'value': datetime.datetime(2015, 6, 2, 12, 18, 26, 279909)}, 'order_line': {'fieldname': 'Order Lines', 'value': sale.order.line(16649, 16651, 16650)}}]
+
+
+
+
+NOTE this module was named `base_jsonify` till version 14.0.1.5.0.
```

Comment:
It seems more logical if this notice resides just before the new doc section you're introducing. Besides - `README.rst` is a generated file - it's [DESCRIPTION.rst](https://github.com/OCA/server-tools/blob/14.0/jsonifier/readme/DESCRIPTION.rst) you should be modifying, or rather `USAGE.rst`, which does not exist yet.

Reply:
Moved to Usage.rst